client/js/game.js

```diff
@@ -6849,11 +6849,12 @@ define(['infomanager', 'bubble', 'renderer', 'map', 'animation', 'sprite', 'tile
             },
 
             findVisibleAnimatedTiles: function(animatedTiles) {
+                let scene = this.map.getCurrentScene(this.player);
                 let visibleAnimatedTiles = [];
                 const animatedTilesLength = animatedTiles.length;
                 for (let i = 0; i < animatedTilesLength; i++) {
                     let tile = animatedTiles[i];
-                    if (this.camera.isVisiblePosition(tile.x, tile.y, 2)) {
+                    if (this.camera.isVisiblePosition(tile.x, tile.y, 20) && (!scene || !scene.isOutOfBounds(tile.x, tile.y))) {
```

Comment:
@balkshamster  This line is probably the issue. It is needed to render the impact of lgihts that are not in the camera view...but their radius is. Could probably be optimized